classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,385 @@
+	 */
+	protected static $should_include_js = false;
+
+
+	/**
+	 * Sets the current data store object into `store->action` and initialises the object.
+	 */
+	public function __construct() {
+		self::$should_include_js = true;
+
+		$this->columns = array(
+			'hook'   => __( 'Hook', 'action-scheduler' ),
+			'status' => __( 'Status', 'action-scheduler' ),
+			'args'   => __( 'Arguments', 'action-scheduler' ),
+			'group'  => __( 'Group', 'action-scheduler' ),
+			'recurrence' => __( 'Recurrence', 'action-scheduler' ),
+			'scheduled'  => __( 'Scheduled Date', 'action-scheduler' ),
+			'claim_id'   => __( 'Claim ID', 'action-scheduler' ),
+			'comments'   => __( 'Log', 'action-scheduler' ),
+		);
+
+		$this->row_actions = array(
+			'hook' => array(
+				'run' => array(
+					__( 'Run', 'action-scheduler' ),
+					__( 'Process the action now as if it were run as part of a queue', 'action-scheduler' ),
+				),
+			),
+		);
+
+		$this->stores = (object) array(
+			'action' => ActionScheduler_Store::instance(),
+			'log'    => ActionScheduler_Logger::instance(),
+		);
+
+		parent::__construct( array(
+			'singular' => __( 'action-scheduler', 'action-scheduler' ),
+			'plural'   => __( 'action-scheduler', 'action-scheduler' ),
+			'ajax'     => false,
+		) );
+	}
+
+	/**
+	 * Sets up the class. It is executed if `is_admin()` is TRUE.
+	 */
+	public static function init() {
+		add_action( 'admin_footer', __CLASS__ . '::maybe_include_assets' );
+		add_action( 'admin_enqueue_scripts', __CLASS__ . '::register_javascript' );
+	}
+
+	/**
+	 * Include our assets files if needed.
+	 *
+	 * Instead of loading our javascript everywhere in the WP-Admin, this method will make sure
+	 * our assets files are loaded only if this class is constructed.
+	 *
+	 * This method will be called in the admin_footer and it will print the CSS we need right away
+	 * and will queue our javascript file.
+	 *
+	 * Our javascript/css does the following things:
+	 *   1. Opens the logs in a modal.
+	 *   2. Loads the CSS for jquery-ui (used in the modals):
+	 */
+	public static function maybe_include_assets() {
+		if ( ! self::$should_include_js ) {
+			return;
+		}
+		wp_enqueue_script( 'action-scheduler' );
+		wp_print_styles( 'wp-jquery-ui-dialog' );
+	}
+
+	/**
+	 * Registers our javascript library and its dependencies.
+	 */
+	public static function register_javascript() {
+		wp_register_script( 'action-scheduler', plugins_url( 'action-scheduler.js', dirname( __FILE__ ) ), array(
+			'jquery',
+			'jquery-ui-core',
+			'jquery-ui-dialog',
+		), '1.0', true );
+	}
+
+	/**
+	 * Returns the recurrence of an action or 'Non-repeating'. The output is human readable.
+	 *
+	 * @param ActionScheduler_Action $item
+	 *
+	 * @return string
+	 */
+	protected function get_recurrence( $item ) {
+		$recurrence = $item->get_schedule();
+		if ( method_exists( $recurrence, 'interval_in_seconds' ) ) {
+			return self::human_interval( $recurrence->interval_in_seconds() );
```

Comment:
Where is `self::human_interval()` defined? I don't see it in `PP_List_Table` or `ActionScheduler_ListTable`.

Reply:
Fixed already